documentation/modules/rn-2.6.adoc

```diff
@@ -0,0 +1,94 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Release_notes/master.adoc
+
+[id="rn-26_{context}"]
+= {project-full} 2.6
+
+Migrate virtual machines (VMs) from VMware vSphere or {rhv-full} or {osp} to {virt} with {the-lc} {project-first}.
```

Comment:
```suggestion
Migrate virtual machines (VMs) from VMware vSphere, {rhv-full} or {osp} to {virt} with {the-lc} {project-first}.
```

Reply:
You have removed this correction. Why?

Reply:
Changed text to Red Hat standards according to Vale